package.json

```diff
@@ -159,6 +159,7 @@
     "request-promise": "4.2.4",
     "require-hijack": "1.2.1",
     "rocambole": "0.7.0",
+    "selenium-webdriver": "4.0.0-alpha.1",
```

Comment:
@erwinmombay @estherkim Is this needed in the main `package.json`? It's already installed here:

https://github.com/ampproject/amphtml/blob/a5be6184fb8c019dd7af9b940df892e543413c00/build-system/tasks/e2e/package.json#L12

Reply:
Oops no it's not needed there.